.github/workflows/test_cpp.yml

```diff
@@ -224,27 +239,33 @@ jobs:
         include:
           - name: C++14
             flags: -DCMAKE_CXX_STANDARD=14
+            run-on-presubmit: true
           - name: C++17
             flags: -DCMAKE_CXX_STANDARD=17
+            run-on-presubmit: false
           - name: C++20
             flags: -DCMAKE_CXX_STANDARD=20
+            run-on-presubmit: true
 
-    name: Linux CMake GCC ${{ matrix.name }}
+    name: ${{ (!matrix.run-on-presubmit && inputs.test-type == 'presubmit') && '[SKIPPED]' || '' }} Linux CMake GCC ${{ matrix.name }} ${{ !matrix.run-on-presubmit && '(Continuous)' || '' }}
     runs-on: ubuntu-latest
     steps:
       - name: Checkout pending changes
+        if: ${{ matrix.run-on-presubmit || inputs.test-type == 'continuous' }}
```

Comment:
If someone mispelled "continuous" this could just silently do nothing.  Making a new boolean input might make this less error-prone, but would unfortunately still have its own overhead (I'm not convinced which is better)

Reply:
You could make it a string that you treat purely as a boolean.   An empty string will evaluate to false, and if nobody cares *what* the value is for "true" it works out fine